examples/packages/npm_install_dup.spec.js

```diff
@@ -0,0 +1 @@
+require('./test_version').test('1.2.0');
```

Comment:
instead of checking in the extra file, can you create it with a `genrule` that copies from the other during the build?

Reply:
just removed the two extra version test. not needed.